Add 2 Projects to "Real-world Projects" section (#9)
iterative/awesome-iterative-projects: readme.md

* [LensKit/lk-demo-experiment](https://github.com/lenskit/lk-demo-experiment): Demo DVC experiment [pipeline (DAG)](https://dvc.org/doc/user-guide/glossary#pipeline-DAG) using multiple public datasets, preprocessing & training, and Jupyter notebooks.
* [ModelOriented/MAIR](https://github.com/ModelOriented/MAIR): Monitoring impact of AI regulations with a DVC pipeline.
* [Kaggle-Titanic-DVC](https://dagshub.com/kingabzpro/kaggle-titanic-dvc): Survival analysis DVC experiment.

> * [DVC with PyCaret & FastAPI](https://github.com/tezansahu/dvc-pycaret-fastapi-demo): Demo about using DVC (for data & model tracking) along with PyCaret (for efficient experimentation) & FastAPI (for model deployment)

Reviewer: ```suggestion
* [DVC with PyCaret & FastAPI](https://github.com/tezansahu/dvc-pycaret-fastapi-demo): End-to-end demo of data & model tracking (DVC), efficient experimentation (PyCaret) & model deployment (FastAPI)
```

> * [VQA-With-Multimodal-Transformers](https://github.com/tezansahu/VQA-With-Multimodal-Transformers): Experimentation with various multimodal [transformer models](https://huggingface.co/docs/transformers/index) for the Visual Question Answering task on [DAQUAR Dataset](https://www.kaggle.com/tezansahu/processed-daquar-dataset), all tracked with DVC.

Reviewer: ```suggestion
* [VQA-With-Multimodal-Transformers](https://github.com/tezansahu/VQA-With-Multimodal-Transformers): Visual Question Answering task on the [DAQUAR Dataset](https://www.kaggle.com/tezansahu/processed-daquar-dataset) using multimodal [transformer models](https://huggingface.co/docs/transformers/index) with an experiment pipeline tracked in DVC Studio.
```
